.build/build-resolver.xml

```diff
@@ -279,6 +288,15 @@
             <url url="${lib.download.base.url}/lib/sigar-bin/libsigar-x86-linux.so"/>
             <url url="${lib.download.base.url}/lib/sigar-bin/libsigar-x86-solaris.so"/>
         </get>
+
+
+        <!-- Amazon Corretto Crypto Providers for various architectures -->
+        <get src="${artifact.github.release}/corretto/amazon-corretto-crypto-provider/releases/download/2.2.0/AmazonCorrettoCryptoProvider-2.2.0-linux-x86_64.jar"
+             dest="${local.repository}/software/amazon/cryptools/AmazonCorrettoCryptoProvider/2.2.0/AmazonCorrettoCryptoProvider-2.2.0-linux-x86_64.jar"
+             usetimestamp="true" quiet="true" skipexisting="true"/>
+        <get src="${artifact.github.release}/corretto/amazon-corretto-crypto-provider/releases/download/2.2.0/AmazonCorrettoCryptoProvider-2.2.0-linux-aarch_64.jar"
+             dest="${local.repository}/software/amazon/cryptools/AmazonCorrettoCryptoProvider/2.2.0/AmazonCorrettoCryptoProvider-2.2.0-linux-aarch_64.jar"
+             usetimestamp="true" quiet="true" skipexisting="true"/>
```

Comment:
can we not resolve and pull by maven coordinates ?

Reply:
probably possible, yeah...

Reply:
@michaelsembwever look, I am not going to do this. This is hell in Ant ... The thing is that we need to check if a file is present locally and if it is not we are going to call mvn plugin which downloads it, I tried to add the target to that but the logic around it is just ... meh. This is super straightforward.

Reply:
yes it's a pita.

it is done like
```

        <resolve>
            <dependencies>
                <dependency groupId="software.amazon.cryptools" artifactId="AmazonCorrettoCryptoProvider" version="2.2.0" classifier="linux-x86_64" />
                <dependency groupId="software.amazon.cryptools" artifactId="AmazonCorrettoCryptoProvider" version="2.2.0" classifier="linux-aarch_64" />
            </dependencies>
        </resolve>
```